src/containers/Dashboard/Tutorials/Tutorial.js

```diff
@@ -1,14 +1,54 @@
 import React, { Component, Fragment } from "react";
-import {
-  INTERVIEW_PREP_KIT,
-  STATISTICS,
-  JAVASCRIPT,
-  CODE
-} from "../../../constants";
+// import {
```

Comment:
remove commented code all over the files you hve worked on

Reply:
Done